### ISSUE TITLE
Linux build does not produce bin folder any longer

### ISSUE THREAD

**Issue by kakobrekla** [hirschmann/nbfc#157]
Hello,

I am trying to compile the latest version on linux according to the published [instructions](https://github.com/hirschmann/nbfc/wiki/How-to-build-NBFC). Bulding is successful, however it does not produce the expected result in `nbfc/Linux/bin/ReleaseLinux`. 
After a lot of fiddling without any change, I tried and succeed compiling previous versions which produced the expected result.

These are the dirs after successful compilation:

`find */Linux/ -type f`

```nbfc-1.4.2/Linux/start-nbfcservice.sh
nbfc-1.4.2/Linux/stop-nbfcservice.sh
nbfc-1.4.2/Linux/bin/ReleaseLinux/Plugins/StagWare.Plugins.ECLinux.dll
nbfc-1.4.2/Linux/bin/ReleaseLinux/Plugins/StagWare.Hardware.LPC.dll
nbfc-1.4.2/Linux/bin/ReleaseLinux/Plugins/StagWare.Plugins.FSTemperatureMonitor.dll
nbfc-1.4.2/Linux/bin/ReleaseLinux/start-nbfcservice.sh
nbfc-1.4.2/Linux/bin/ReleaseLinux/clipr.dll
nbfc-1.4.2/Linux/bin/ReleaseLinux/ec-probe.exe
nbfc-1.4.2/Linux/bin/ReleaseLinux/nbfc.exe.config
nbfc-1.4.2/Linux/bin/ReleaseLinux/StagWare.FanControl.Service.dll
nbfc-1.4.2/Linux/bin/ReleaseLinux/StagWare.FanControl.Configurations.dll
nbfc-1.4.2/Linux/bin/ReleaseLinux/Configs/Sony Vaio SVE1713Y1E.xml
 ... snip ...
nbfc-1.4.2/Linux/bin/ReleaseLinux/Configs/Asus Vivobook S400CA.xml
nbfc-1.4.2/Linux/bin/ReleaseLinux/StagWare.FanControl.dll
nbfc-1.4.2/Linux/bin/ReleaseLinux/nbfc.exe
nbfc-1.4.2/Linux/bin/ReleaseLinux/stop-nbfcservice.sh
nbfc-1.4.2/Linux/bin/ReleaseLinux/NbfcService.exe
nbfc-1.4.2/Linux/bin/ReleaseLinux/NbfcService.exe.config

nbfc-1.4.4-beta/Linux/start-nbfcservice.sh
nbfc-1.4.4-beta/Linux/continue-nbfcservice.sh
nbfc-1.4.4-beta/Linux/pause-nbfcservice.sh
nbfc-1.4.4-beta/Linux/nbfc-sleep.service
nbfc-1.4.4-beta/Linux/stop-nbfcservice.sh
nbfc-1.4.4-beta/Linux/nbfc.service
nbfc-1.4.4-beta/Linux/bin/ReleaseLinux/Plugins/StagWare.Plugins.ECLinux.dll
nbfc-1.4.4-beta/Linux/bin/ReleaseLinux/Plugins/StagWare.Hardware.LPC.dll
nbfc-1.4.4-beta/Linux/bin/ReleaseLinux/Plugins/StagWare.Plugins.FSTemperatureMonitor.dll
nbfc-1.4.4-beta/Linux/bin/ReleaseLinux/Plugins/StagWare.Plugins.ECSysLinux.dll
nbfc-1.4.4-beta/Linux/bin/ReleaseLinux/start-nbfcservice.sh
nbfc-1.4.4-beta/Linux/bin/ReleaseLinux/clipr.dll
nbfc-1.4.4-beta/Linux/bin/ReleaseLinux/ec-probe.exe
nbfc-1.4.4-beta/Linux/bin/ReleaseLinux/nbfc.exe.config
nbfc-1.4.4-beta/Linux/bin/ReleaseLinux/StagWare.FanControl.Service.dll
nbfc-1.4.4-beta/Linux/bin/ReleaseLinux/continue-nbfcservice.sh
nbfc-1.4.4-beta/Linux/bin/ReleaseLinux/pause-nbfcservice.sh
nbfc-1.4.4-beta/Linux/bin/ReleaseLinux/StagWare.FanControl.Configurations.dll
nbfc-1.4.4-beta/Linux/bin/ReleaseLinux/Configs/Sony Vaio SVE1713Y1E.xml
 ... snip ...
nbfc-1.4.4-beta/Linux/bin/ReleaseLinux/Configs/HP OMEN Notebook PC 15.xml
nbfc-1.4.4-beta/Linux/bin/ReleaseLinux/StagWare.FanControl.dll
nbfc-1.4.4-beta/Linux/bin/ReleaseLinux/nbfc.exe
nbfc-1.4.4-beta/Linux/bin/ReleaseLinux/stop-nbfcservice.sh
nbfc-1.4.4-beta/Linux/bin/ReleaseLinux/NbfcService.exe
nbfc-1.4.4-beta/Linux/bin/ReleaseLinux/NbfcService.exe.config

nbfc-1.4.5-beta/Linux/start-nbfcservice.sh
nbfc-1.4.5-beta/Linux/continue-nbfcservice.sh
nbfc-1.4.5-beta/Linux/pause-nbfcservice.sh
nbfc-1.4.5-beta/Linux/nbfc-sleep.service
nbfc-1.4.5-beta/Linux/stop-nbfcservice.sh
nbfc-1.4.5-beta/Linux/nbfc.service

nbfc-1.5/Linux/start-nbfcservice.sh
nbfc-1.5/Linux/continue-nbfcservice.sh
nbfc-1.5/Linux/pause-nbfcservice.sh
nbfc-1.5/Linux/nbfc-sleep.service
nbfc-1.5/Linux/stop-nbfcservice.sh
nbfc-1.5/Linux/nbfc.service
```

So in summary, versions 1.4.2. and 1.4.4. produce the `bin` direcotry containing the result of  the build, 1.4.5. and 1.5. do not. This was tested on a fresh install:

```
cat /etc/issue
Ubuntu 16.10

mono -V
Mono JIT compiler version 4.6.2 (Stable 4.6.2.7/08fd525 Mon Nov 21 12:08:40 UTC 2016)
Copyright (C) 2002-2014 Novell, Inc, Xamarin Inc and Contributors. www.mono-project.com
	TLS:           __thread
	SIGSEGV:       altstack
	Notifications: epoll
	Architecture:  amd64
	Disabled:      none
	Misc:          softdebug 
	LLVM:          supported, not enabled.
	GC:            sgen

```

Any suggestions?

**Comment by hirschmann** [comment]
Fixed with 524ecf1cf95a261e574e6c039c97ed8206e50e56
Thanks for the report!

**Comment by kakobrekla** [comment]
Thank you for the fix -- confirmed working.
